gamebook/game_manager.py

```diff
@@ -23,6 +21,8 @@ def start(self, current_scene):
         self.current_scene = current_scene
         while True:
             next_scene_name = self.run_scene()
+            if next_scene_name == "":
```

Comment:
Maybe raise a custom exception that you've reached the end? it will be more understandable.